README.md

```diff
@@ -69,7 +69,9 @@ Logflare is using a SQL parser from sqlparser.com. To set this up on your dev ma
 ```elixir
 iex> LogflareLogger.info("testing log message")
 ```
-
+### Using Docker
+1. Build images with `docker compose build --build-arg=magic_cookie=$(cat /dev/urandom | LC_ALL=C tr -dc 'a-zA-Z0-9' | fold -w 50 | head -n 1)`
```

Comment:
as above, we don't need to set the magic cookie at build time.

Also, we can use a simple value like `mycookie` for the cookie as it is for illustration purposes.

the dcker compose file would also serve as an example.

Reply:
will change